app/routes.py

```diff
@@ -1,2 +1,53 @@
-from flask import Blueprint
+from flask import Blueprint, jsonify, abort, make_response, request
+from app import db
+from app.models.planet import Planet
 
+planet_bp = Blueprint("planets", __name__, url_prefix="/planets")
+
+def validate_planet(id):
+    try:
+        id = int(id)
+    except:
+        abort(make_response({"message": f"planet '{id}' is invalid"}, 400))
+
+    planet = Planet.query.get(id)
+
+    if not planet:
+        abort(make_response({"message": f"Planet {id} not found"}, 404))
+
+    return planet
+
+@planet_bp.route("", methods=["GET", "POST"])
+def handle_planets():
+    if request.method == "POST":
+        request_body = request.get_json()
+        new_planet = Planet(name=request_body["name"], 
+                            description=request_body["description"],
+                            flag=request_body["flag"])
+        db.session.add(new_planet)
+        db.session.commit()
+        return make_response(f"planet {new_planet.name} successfully created!", 201)
```

Comment:
👀 Make sure to `jsonify` even plain string responses, otherwise flask will return an HTML response rather than JSON.